### PR TITLE
Adds ability to toggle multiple ports at once

### PR DESCRIPTION
**Pull request by ch-iara**
New save button on port assignment page will now send
the port assignments all at once and reboot once,
rather than having to do one at a time and reboot
with each change

goes with https://github.com/althea-net/althea_rs/pull/660

### CODE REVIEW

File: src/RouterSettings/Ports.js [hunk 1]
```diff
@@ -44,14 +44,33 @@ const Ports = () => {
...
   useInterval(getInterfaces, 5000);
 
-  let setInterfaceMode = (iface, mode) => {
-    setSelected(iface);
-    if (mode === "Wan") {
+  // note (TODO) iface has been set on backend to accept a Vec<string> of interfaces, and mode a Vec<InterfaceMode>
+  function setInterfaceMode() {
+    //   //TODO change this back to before- but clear selected. this fn will now be called on save button
+    //   //setSelected(selected.push(iface));
+    //   //if (new_mode === "Wan") {
```
Comment (jkilpatr) [inline src/RouterSettings/Ports.js:51]: not sure if these todo's need to be resolved

File: src/RouterSettings/Ports.js [hunk 2]
```diff
@@ -62,9 +81,11 @@ const Ports = () => {
     setError(null);
     setConfirming(false);
     let unexpectedError = false;
+    let count = 0; // to keep track of which set we are on in order to send error info if one occurs
```
Comment (jkilpatr) [inline src/RouterSettings/Ports.js:84]: can you expand on this comment? I'm having some trouble following the logic.

**Review by jkilpatr — COMMENTED** [review]
Left a few questions

File: src/RouterSettings/Ports.js (hunk 1 above)
Comment (ch-iara) [reply src/RouterSettings/Ports.js:51]: Whoops, forgot to delete the last of the TODOs. fixed

File: src/RouterSettings/Ports.js (hunk 2 above)
Comment (ch-iara) [reply src/RouterSettings/Ports.js]: This... was left over from when I had been trying to send this command in a loop and decided that wasn't the best way, oops. It's been changed to do what I had actually intended now

**Review by jkilpatr — APPROVED** [review]

**Pull request merged**